app/templates/main/userProfile.html

```diff
@@ -603,7 +603,9 @@ <h6 align="left">Reason for ban:</h6>
             <p align="left" id="banNote">Why the student was banned</p>
           </div>
           <div id="banEndDate">
-            <h6 align="left" class="d-inline-flex" id="unbanVolunteerEndDate">End Date: &nbsp<input id="banEndDatepicker" type="date"></h6>
+            <h6 align="left" class="d-inline-flex" id="unbanVolunteerEndDate">End Date: &nbsp
+              <input id="banEndDatepicker" readonly='readonly'  type= {% if isChrome == True %} "text" placeholder="mm/dd/yyyy" {% else %} "text" placeholder="mm/dd/yyyy" {% endif %}>
```

Comment:
The `== True` part is redundant. Also, what is the difference between the Chrome case and the Firefox case?